homeassistant/components/switcher_kis/light.py

```diff
@@ -35,18 +35,18 @@ async def async_setup_entry(
     def async_add_light(coordinator: SwitcherDataUpdateCoordinator) -> None:
         """Add light from Switcher device."""
         entities: list[LightEntity] = []
+        lights = cast(SwitcherLight, coordinator.data.light)
```

Comment:
```suggestion
        lights = cast(SwitcherLight, coordinator.data)
```

Reply:
I changed that to number_of_lights. 
Should I name it number_of_switcher_lights?